### PR TITLE
Fuck it, we are rebasing to Eris

### PR DESCRIPTION
**Pull request by BobJoga**
##  About The Pull Request

TLDR stealing a bunch of Eris sprites, focusing on turfs, machinery and structure, other stuff may come later. This is far from done, but here are screencaps of my progress:

image
First screencap, no idea what i was doing, just happy to get it working somewhat.

image
Second screencap, made a lot of progress but it's still a bit fucked, the worst offender being that security still has ironhammer door even though ours is a bunch of soviet fucks.

image
Third screencap, captain's room looks especially nice even though i fucked up on reinforced walls (you can see a weird squiggly line on them where there should not be one). Showcases how the new wood floor i also stole from Eris meshes in, too.

image
Fourth screencap... maintenance is a bit too high techy for my tastes, and the doors are too piss yellow. Me and JungleRat have since changed that.

image
Fifth screencap, here is the new improved maint door. A bit more shitty as it should be, but JungleRat will make it even more edgy. Note how the plating is suddenly gone - i don't know what the fuck i just did, it was supposed to be the same plating but shifted to be centered.

image
Sixth screencap... these windows i did myself otherwise the old ones would stick out way too much. As you might have noticed, they are horrible, thank you!

image
Seventh screencap... the corridor currently looks horrible because i have yet to prepare proper public airlocks, even less so in regards to glass ones so... they look like vomit. I guess.

image
Eighth screencap. It's not like there is an escape, but here's how the plating looks now, along with... escape airlocks. Which really look more like what security airlocks should look like... i'll probably paint these darker or something. New plating is now looking satisfactory though!

image
Ninth (?) screencap. If you think this was going too smoothly so far, here's a trainwreck - pods look awful. God awful. Like, really shit!

image
Tenth screencap. Material walls and doors! I think these look fine, i just went too soft on coloring. Otherwise, yeah, good.

image
Eleventh screencap. I did hit the mark well with uranium though. Nice and radioactive, mhm, though the material walls probably need to be roughened up a bit, looking a bit too soft for my tastes, but not bad.

image
Twelfth... (?) screencap. Maintenance looks good. Maybe too good.

image
Thirteenth, and honoring the number, it's not that great. More shitty window, drawn by yours truly (It's a tinted window).

image
Fourteenth screencap. Here's more shitty windows. Tbh they might look fine once everything is set and done, starting to warm up to them. These are reinforced ones.

image
Fifteenth screencap. Engineering doors recolored to fit our engineering standards, thanks to JungleRat. Looking nice and bright aside from the Ironhammer security lounge!

image
Sixteenth. Well, the pods worked fine in this one edge case actually. Not too bad, i probably just need to add some more darker finish to the corner thingies, maybe smooth them down so they're 100% diagonal.

image
Seventeenth. Reinforced walls are blending together fine, aside from the fact they... are failing to connect at places... I'll fix that.

image
Eighteenth screencap. Here we can see the consequences of using a "country girls make do" overlay instead of a proper, handcrafted one.

image
Nineteenth screncap. I finally fixed public glass doors. Looking kinda fresh. I fixed the stuck emergency lights now.

image
Twentieth screencap... oh god this looks horrible. What happened here is, Eris uses low walls - We don't have low walls and i don't feel like editing every map to have them, so i just made my own window sprites. And it looks awful, garbage.

image
hmmmm catwalke....

image
The windows are... getting there.

image
fuck

image
pyramid bar

image
idk the lighting is kinda making everything look shit

image
this isn't looking too good

image
ok making the windows smooth with airlocks actually made things worse, maybe if i remove firelocks from the list

image
its looking good for real now

image
oh damn its fresh as fuck

image
Solved the lighting issue by changing planes around but now we get this.

image
Almost ready for deployment. 🇺🇸 

image
Walls have proper brims now. This is probably the least update to this, i hope it does get merged however.

**General notes:**
Airlocks are borked and always have the emergency light turned on. I didn't have the patience to do that as i already had to fiddle a lot with airlocks to make them use the overlays instead of static sprites. I will fix that.

The sprites are messy and full of issues, some of them i have yet to resolve, but please do point out every little mistake because i am a bit nearsighted.

There are some unused assets like the onestar walls. No idea what i'll use them for.

I'm not a spriter i only started doing this for fun, and i'm kind of stupid so there might be many stupid rookie mistakes here i simply have failed to catch because i'm an idiot. Yeah. In fact i kinda hate spriting, any help is good.

I have yet to do fulltile windows but do not fret, they will be freshened up too.

I have to do other random structures, mostly bar-related of course, but they will be done eventually.

Some floors need to be lightened up a bit, but i feel this will be easier since decals do most decoration and colorwise most floors fit in fine.

**Random thoughts**
Next on the list... porting the 2nd amendment and all guns from Eris. Or more realistically deepmaint. But why not shoot for both?

And Bobmed Director's cut, with the dialysis chamber DLC included.

## Why It's Good For The Game

Things are for the moment looking quite trashy, but from the screencaps alone i do think this can go forward to make our server look much much better. A lot of stuff will be quite whiplashy for a lot of time but, this can be solved eventually, and this shakes off some of the more arcadey TG feel out of our code... maybe Bobmed now maintainers? 🙄 Visual changes alone won't solve everything. They do help though god damn.

## Changelog
:cl: CEV Eris and Hyperstation, i won't put my name here cause i barely did any work
imageadd: Everything looks like Eris now. Because Eris does everything better in general. Shoutouts to CEV Eris, really good server please play there. No i'm not being paid they're just good and really do deserve a sponsorship just because of the spritework they did. Thanks.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
Already looks like a huge improvement. Very nice work.

**Comment by BobJoga** [comment]
> Already looks like a huge improvement. Very nice work.

I did put a lot of effort into this but please do take in mind that this is really me just stealing square pegs and fitting them onto our round hole so yeah, credits go to the CEV Eris coders and spriters because they are kinda based.

**Comment by Investigator77** [comment]
This actually looks nicer than I initially thought. As a spriteman, has my support

**Comment by CoreFlare** [comment]
This is going to give me codebase whiplash.
But if we get the guns too then this will be hilarious.

**Comment by ShadeAware** [comment]
### YOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO HOLY SHIT LETS FUCKING GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by BobJoga** [comment]
> This is going to give me codebase whiplash.
> But if we get the guns too then this will be hilarious.

Yes it will and i will not pretend for a single moment that everything will look clean and fitting at first.

**Comment by XyvMeph** [comment]
I love everything about this except the windows, if only because I think they're a bit jarring against the rest of the sprites. If they were more transparent that'd be Ideal. With glass, the reflective pieces should be towards the middle, and not rimmed in a bright color.

**Comment by Owai-Seek** [comment]
Not really fond of the uncovered floor tiles / maintenance floor tiles. They give off an abductor vibe.

The windows don't look -bad- but they seem a bit too bright when compared everything else.

A lot of stuff looks -really- weird with these walls, which is an easy solve I suppose, just grabbing the sprites for those things. (Air alarms, extinguishers, fire alarms, ect.)

I do like the airlocks and walls, assuming the wall stuff gets brought on to match it.

**Comment by Azarak** [comment]
Looks pretty damn good. Only gripe with is the missing sprites aswell as the plating sprite, in Eris, in maintenance you usually walk over catwalks and even then I think they've got a different sprite for plating that's under the catwalks too? Not sure

**Comment by Akrilla** [comment]
:pog:

**Comment by BobJoga** [comment]
> I love everything about this except the windows, if only because I think they're a bit jarring against the rest of the sprites. If they were more transparent that'd be Ideal. With glass, the reflective pieces should be towards the middle, and not rimmed in a bright color.

I have yet to port fulltile windows, but i'll take this into account for my own shitty windows.

**Comment by BobJoga** [comment]
> Not a fan of the maintenance tiles, because they give off an abductor vibe, but you already say they're kinda iffy yourself.
> 
> The windows don't look -bad- but they seem a bit too bright when compared everything else.
> 
> A lot of stuff looks -really- weird with these walls, which is an easy solve I suppose, just grabbing the sprites for those things. (Air alarms, extinguishers, fire alarms, ect.)
> 
> I do like the airlocks and walls, assuming the wall stuff gets brought on to match it.

Yes a lot of structural stuff will be brought over. I'm starting with turfs and airlocks because they're the hardest, and most important, to solve.

**Comment by BobJoga** [comment]
> Looks pretty damn good. Only gripe with is the missing sprites aswell as the plating sprite, in Eris, in maintenance you usually walk over catwalks and even then I think they've got a different sprite for plating that's under the catwalks too? Not sure

The problem with eris maint is, a lot of it doesn't actually use plating as you said, it's mostly catwalks and random floor tiles. I cannot do much about that.

**Comment by ShadeAware** [comment]
This is starting to look pretty damn fucking incredible

**Comment by ShadeAware** [comment]
My only gripe is that I think the walls should actually connect to the airlocks like they do windows, along with the sprites for wall objects,  but other than that its 👀❤

**Comment by BobJoga** [comment]
> This is starting to look pretty damn fucking incredible

honestly the more i work on this the more i realize this might be wasted effort

image

i don't know man shit getting a bit xie hua piao piao piao

i dont know if i can make airlocks connect to walls proper because simply put i'll need to add all subtypes to the cansmoothwith list, via shitcode or manually (not happening)

**Comment by BobJoga** [comment]
i did it reddit, the final hurdle of bullshit layering was overcome
image
the fucking disposal pipe is gone

**Comment by werewolfboyfriend** [comment]
Would Bay's take on the windows in this perspective be worth mixing in? 
Capture
Either way.. windoors would look a ton better if you could get the walls/windows to sit flush with them like they do with doors.

**Comment by BobJoga** [comment]
> Would Bay's take on the windows in this perspective be worth mixing in?
> Capture
> Either way.. windoors would look a ton better if you could get the walls/windows to sit flush with them like they do with doors.

I tried that but in my honest opinion it looked worse, way worse, and it made no sense for them to connect with windoors since it caused hilarity when a windoors facing away from the wall "connected with it". I am a man of action rather than words, so i'll send screenshots soon to prove how horrid it looks when i get the windoors to "sit flush" with walls. If i want to fix this, it would require an enormous amount of shitcode, making so that only north and south facing windoors connect. Which is **bad**.

**Comment by ShadeAware** [comment]
### BROOOOOOOOOOOOOO THIS LOOKS SO FUCKING FRESH

**Comment by BobJoga** [comment]
> Would Bay's take on the windows in this perspective be worth mixing in?

image
No, it wouldn't. This doesn't look right, now... does it?

**Comment by ShadeAware** [comment]
This is looking fresh as fuck. Although, I have a few bullet points I'd like to voice:
1. Maint tiles just don't look right, kind of... too techy, if you catch my drift.
2. We could use some more Eris Floors to help cement the look and feel. The walls look **great** but floors would really help sell the whole thing even more.
image

**Comment by ShadeAware** [comment]
Also, false walls don't have a new texture.
image

**Comment by ShadeAware** [comment]
Rusted walls are looking jank aswell
image

**Comment by BobJoga** [comment]
> This is looking fresh as fuck. Although, I have a few bullet points I'd like to voice:
> 
> 1. Maint tiles just don't look right, kind of... too techy, if you catch my drift.
> 2. We could use some more Eris Floors to help cement the look and feel. The walls look **great** but floors would really help sell the whole thing even more.
>    image

1. yes but old plating looks even worse and even more unfitting and i have no artistic ability to make decent fucking plating sprites
2. plan to do it in a separate PR because we use vg decals and adjusting those to fit in will be cock and ball torture

**Comment by BobJoga** [comment]
> Rusted walls are looking jank aswell
> image

i'll have to scrap my own pile of junk sprite for rust walls because no one has 3/4ths rust so wooo

**Comment by BobJoga** [comment]
> Also, false walls don't have a new texture.
> image

fixed

**Comment by CameronWoof** [comment]
Looking good. Only complaints are the new security airlocks and thindows.
image

Really washed-out, not very nice to look at imo. And the new thindows are odd to me--they're not in a different perspective and look like codersprite versions of the ones we already had. Why change them?

image

This is how they're used most often, and they clash with the windoors pretty bad.

**Comment by BobJoga** [comment]
ok so people have been complaining about the new walls being too bright

im not gonna say anything besides this

image

**Comment by BobJoga** [comment]
> Looking good. Only complaints are the new security airlocks and thindows.
> image
> 
> Really washed-out, not very nice to look at imo. And the new thindows are odd to me--they're not in a different perspective and look like codersprite versions of the ones we already had. Why change them?
> 
> image
> 
> This is how they're used most often, and they clash with the windoors pretty bad.

i just picked the shitty bland directional window sprites eris has, best i can get for now
the codersprites were simply because at a distance it looked better since it's more colored but eh whatever throw it in the trash
i did fix the window ovelays though, but i don't want to recolor the security airlocks because it's such a hard balance to reach with the coloring without making sec look like the soviet union and i'm so fucking tired

for now, what you see is what you get pretty much

**Comment by BobJoga** [comment]
image

So -7 brightness seems to be the sweetspot so walls dont get *that* bright, but i will still have to do a fuckton of manual work to get it to this brightness once i get back working on this. I vastly underestimated how hellish of a task remaking years of sprites actually is, even when most of it is stolen.

**Comment by Owai-Seek** [comment]
This should probably be converted to a draft.

**Comment by BobJoga** [comment]
> This should probably be converted to a draft.

i'm actually still working on it, let me update the title. i'm just tired but it should be ready to be merged really.

**Comment by werewolfboyfriend** [comment]
I was having a look at this in a local copy, 'cause I'm excited, and I do have one suggestion: Eris floors would make this feel a lot less out of place, especially Eris tiles and wood. Underplating might work better for plating here, too, though it's super dark.

**Comment by BobJoga** [comment]
> I was having a look at this in a local copy, 'cause I'm excited, and I do have one suggestion: Eris floors would make this feel a lot less out of place, especially Eris tiles and wood. Underplating might work better for plating here, too, though it's super dark.

ehhhhhh idk man doesn't really fit well at all

**Comment by BobJoga** [comment]
> I was having a look at this in a local copy, 'cause I'm excited, and I do have one suggestion: Eris floors would make this feel a lot less out of place, especially Eris tiles and wood. Underplating might work better for plating here, too, though it's super dark.

actually let me try i think it could look good

**Comment by BobJoga** [comment]
image
ok this is fresh

**Comment by werewolfboyfriend** [comment]
ok that looks pretty sick, a bit bright but still pretty neat

**Comment by BobJoga** [comment]
fuck this pr, we staying with awful ugly sprites
